### PR TITLE
OPSEXP-2311 Bump tags for cassiopeia release

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2311

generated with https://github.com/Alfresco/alfresco-updatecli/pull/10

### CODE REVIEW

File: docker-compose/7.0.N-docker-compose.yml [hunk 1]
```diff
@@ -145,7 +145,7 @@ services:
       - "61616:61616" # OpenWire
       - "61613:61613" # STOMP
   digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
+    image: quay.io/alfresco/alfresco-digital-workspace:2.6.1
```
Comment (gionn) [inline docker-compose/7.0.N-docker-compose.yml:148]:
```suggestion
    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
```

File: docker-compose/7.1.N-docker-compose.yml [hunk 2]
```diff
@@ -145,7 +145,7 @@ services:
       - "61616:61616" # OpenWire
       - "61613:61613" # STOMP
   digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
+    image: quay.io/alfresco/alfresco-digital-workspace:2.6.1
```
Comment (gionn) [inline docker-compose/7.1.N-docker-compose.yml:148]:
```suggestion
    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
```

File: docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -122,7 +122,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: quay.io/alfresco/search-services:2.0.8
+    image: quay.io/alfresco/search-services:2.1.0-A1
```
Comment (gionn) [inline docker-compose/docker-compose.yml:125]:
```suggestion
    image: quay.io/alfresco/search-services:2.0.8.1
```

File: helm/alfresco-content-services/values.yaml [hunk 4]
```diff
@@ -251,7 +251,7 @@ alfresco-search:
   enabled: true
   searchServicesImage:
     repository: quay.io/alfresco/search-services
-    tag: 2.0.8
+    tag: 2.1.0-A1
```
Comment (gionn) [inline helm/alfresco-content-services/values.yaml:254]:
```suggestion
    tag: 2.0.8.1
```

**Review by alxgomz — APPROVED** [review]

**Comment by alxgomz** [comment]
I assume you're planning on releasing in a different PR?

**Pull request merged**